### PR TITLE
Custom secnames

### PR DESCRIPTION
**Pull request by DrTaDa**
Add the option of giving custom seclist_names and secarray_names

### CODE REVIEW

File: bluepyopt/ephys/models.py [hunk 1]
```diff
@@ -96,10 +105,20 @@ def __init__(
...
+            ]
+        else:
+            self.seclist_names = seclist_names
+
+        if secarray_names is None:
+            self.secarray_names = [
+                'soma', 'dend', 'apic', 'axon', 'myelin'
+            ]
+        else:
+            self.secarray_names = secarray_names
```
Comment (arnaudon) [inline bluepyopt/ephys/models.py:122]: what about a check that at least soma/dend/axon exists? Can you run something without any of these?

**Comment by arnaudon** [comment]
+1

File: bluepyopt/ephys/models.py (hunk 1 above)
Comment (DrTaDa) [reply bluepyopt/ephys/models.py:122]: Good question, the other option is to only pass the custom names and to append them to the minimal list.

**Review by wvangeit — APPROVED** [review]

**Comment by DrTaDa** [comment]
@wvangeit , this can be merged.

**Pull request merged**